Jenkinsfile-chart

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 2007-2019, GoodData(R) Corporation. All rights reserved.
+ */
+
+@Library('pipelines-shared-libs')
+import com.gooddata.pipeline.Pipeline
```

Comment:
keep only following content in this file:
```
@Library('pipelines-shared-libs')
import com.gooddata.pipeline.Pipeline

Pipeline.resolve(this, '.gdc-ii-config-chart.yaml').run()
```

and add file in root `.gdc-ii-config-chart.yaml` with following content:
```
charts:
  - 'msf-xk8s'
```

This is a preliminary step before dropping the Jenkinsfiles from repositories